docs/src/main/asciidoc/spring-di.adoc

```diff
@@ -1,28 +1,27 @@
-////
-This guide is maintained in the main Quarkus repository
-and pull requests should be submitted there:
-https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc
-////
-= Quarkus - Quarkus Extension for Spring DI API
+//TODO: need to use product attributes, but also keep compatibility with community ones
 
-include::./attributes.adoc[]
-:extension-status: preview
+// comment out to show community version
+:product-docs:
+:doctype: book
 
-While users are encouraged to use CDI annotations for injection, Quarkus provides a compatibility layer for Spring dependency injection in the form of the `spring-di` extension.
+//Top-Level Assembly
+[id="using-the-quarkus-extension-for-spring-di-api_{context}"]
+= Using the Quarkus Extension for Spring DI API
```

Comment:
```suggestion
= Using the Quarkus extension for Spring Dependency Injection (DI) API
```

Reply:
Thanks:) I updated the title per your suggestion:)